src/modm/platform/i2c/stm32-extended/i2c_master.hpp.in

```diff
@@ -99,12 +99,12 @@ public:
 	 */
 	template<class SystemClock, baudrate_t baudrate=kBd(100), percent_t tolerance=pct(5)>
 	static void
-	initialize()
+	initialize(uint32_t isrPriority = 10u)
```

Comment:
i have a feeling the modm guys probably want the priority as a template parameter -> how is it handled on CAN for example?

Reply:
I will check that. Those are just the commits we made on our local branch as is. What is bothering me the most for now is why the unit tests are failing somewhere we have nothing to do with.

Reply:
it is a parameter passed on constructor. Honestly I would keep as is then.
`modm::platform::Can{{ id }}::initializeWithPrescaler(
		uint16_t prescaler, uint8_t bs1, uint8_t bs2,
		uint32_t interruptPriority, Mode startupMode, bool overwriteOnOverrun)`

Reply:
They also don't have a constructor for the I2c, honestly the best solution is to leave as is